backend/runner/plancheck/test/mysql_statement_type_check.yaml

```diff
@@ -0,0 +1,396 @@
+- statement: |-
+    UPDATE items,
+       (SELECT id FROM items
+        WHERE id IN
+            (SELECT id FROM items
+             WHERE retail / wholesale >= 1.3 AND quantity < 100))
+        AS discounted
+    SET items.retail = items.retail * 0.9
+    WHERE items.id = discounted.id;
+  changetype: 2
+  want: []
+- statement: REPLACE INTO test2 VALUES (1, 'Old', '2014-08-20 18:47:00');
+  changetype: 2
+  want: []
+- statement: |-
+    (SELECT 1 AS result UNION SELECT 2)
+       ORDER BY result DESC LIMIT 1;
+  changetype: 2
+  want: []
+- statement: |-
+    INSERT INTO tbl_temp2 (fld_id)
+    SELECT tbl_temp1.fld_order_id
+    FROM tbl_temp1 WHERE tbl_temp1.fld_order_id > 100;
+  changetype: 1
+  want:
+    - |-
+      {
+        "status":  "WARNING",
+        "title":  "Alter schema can only run DDL",
+        "content":  "\"INSERT INTO tbl_temp2 (fld_id)\nSELECT tbl_temp1.fld_order_id\nFROM tbl_temp1 WHERE tbl_temp1.fld_order_id > 100;\" is not DDL",
+        "code":  402
+      }
+- statement: IMPORT TABLE FROM '/tmp/mysql-files/*.sdi';
+  changetype: 1
+  want: []
+- statement: TABLE a EXCEPT TABLE b;
+  changetype: 1
+  want:
+    - |-
+      {
+        "status":  "WARNING",
+        "title":  "Alter schema can only run DDL",
+        "content":  "\"TABLE a EXCEPT TABLE b;\" is not DDL",
+        "code":  402
+      }
+- statement: EXPLAIN SELECT c->>"$.name" AS name FROM jempn WHERE g > 2;
+  changetype: 1
+  want: []
+- statement: |-
+    CREATE TABLE artists_and_works
+    SELECT artist.name, COUNT(work.artist_id) AS number_of_works
+    FROM artist LEFT JOIN work ON artist.id = work.artist_id
+    GROUP BY artist.id;
+  changetype: 1
+  want: []
+- statement: |-
+    CREATE SPATIAL REFERENCE SYSTEM 4120
+    NAME 'Greek'
+    ORGANIZATION 'EPSG' IDENTIFIED BY 4120
+    DEFINITION
+      'GEOGCS["Greek",DATUM["Greek",SPHEROID["Bessel 1841",
+      6377397.155,299.1528128,AUTHORITY["EPSG","7004"]],
+      AUTHORITY["EPSG","6120"]],PRIMEM["Greenwich",0,
+      AUTHORITY["EPSG","8901"]],UNIT["degree",0.017453292519943278,
+      AUTHORITY["EPSG","9122"]],AXIS["Lat",NORTH],AXIS["Lon",EAST],
+      AUTHORITY["EPSG","4120"]]';
+  changetype: 1
+  want:
+    - |-
+      {
+        "status":  "ERROR",
+        "title":  "Syntax error",
+        "content":  "Syntax error at line 2:5 \nrelated text: EATE SPATIAL REFERENCE SYSTEM 4120\nNAME 'Greek'",
+        "sqlReviewReport":  {
+          "code":  201
+        }
+      }
```

Comment:
Is syntax error expected?

Reply:
I found this statement from mysql [manual](https://dev.mysql.com/doc/refman/8.0/en/create-spatial-reference-system.html), It seems that mysql parser does not support this grammar.

Reply:
You can use another statement or remove it. The SYNTAX ERROR here is useless.